spec/models/question_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
```

Comment:
You can put `--require rails_helper` into your .rspec file and avoid adding this line in each file